### PR TITLE
[droid-hal-configs] Use appropriate key code for device's home key. Contributes to JB#17545

### PR DESCRIPTION
**Pull request by jpetrell**
Home is used in apps to move focus to the beginning, HomePage is the right one to handle navigation to homescreen.

### CODE REVIEW

File: configs/droid.kmap [hunk 1]
```diff
@@ -1,5 +1,4 @@
 keycode 100 = AltGr
-keycode 102 = Home
```
Comment (pvuorela) [inline configs/droid.kmap]: Why is this removed? Affecting home button on external keyboards?

Comment (jpetrell) [reply configs/droid.kmap]: Yes. Apparently there has been complaints in TJC that we broke terminal home key so better remove this one.

Comment (jpetrell) [reply configs/droid.kmap]: Only known place where we use Qt.Key_Home was removed, but kept it for now to be on the safe side.

**Comment by pvuorela** [comment]
LGTM

**Comment by pvuorela** [comment]
Ok, so droid-config.inc does:
/usr/lib/qt5/bin/kmap2qmap %{dcd_common}/configs/droid.kmap %{dcd_common}/configs/us.kmap $RPM_BUILD_ROOT/%{_datadir}/qt5/keymaps/droid.qmap

us.kmap includes bunch of stuff so I think we could delete arrow keys and home from droid.kmap.
